File: easy/bubbleSort.py
def bubbleSort(array):
    n = len(array)
    if n == 1:
        return array

    for i in range(n):
        for j in range(i, n):
            if array[i] > array[j]:
                array[i], array[j] = array[j], array[i]

    return array

File: C++/bubbleSort.cpp
#include <vector>
using namespace std;

vector<int> bubbleSort(vector<int> array) {
  int n = array.size();
  for(int i = 0; i < n; i++){
    for (int j = i; j < n; j++){
      if(array[i] > array[j]){
        swap(array[i], array[j]);
      }
    }
  }
  return array;
}
